pkg/apis/serving/v1alpha1/configuration_validation_test.go

```diff
@@ -195,17 +205,21 @@ func TestConfigurationValidation(t *testing.T) {
 				Name: "do.not.use.dots",
 			},
 		},
-		want: (&apis.FieldError{Message: "Invalid resource name: special character . must not be present", Paths: []string{"metadata.name"}}).
-			Also(apis.ErrMissingField("spec")),
+		want: (&apis.FieldError{
+			Message: "not a DNS 1035 label: [a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')]",
```

Comment:
Something like test/util.go:Error1035 = ...?
When k8s changes this string, it'll be a single place to change.

Reply:
Same for the length.

Reply:
That actually creates an import cycle, and I'm not sure of a good place to put it that could also be shared by autoscaling (we could share it across `pkg/apis/serving/v1alpha1` via a private variable).